client/composables/crumbs.ts

```diff
@@ -1,3 +1,4 @@
+import { ref } from 'vue'
```

Comment:
not sure exaclty why I need to import this here (cause it works without it) but the test would fail